### PR TITLE
Updated Ubuntu Advantage service description

### LINKED ISSUE #2748: Build: UA service description 2018/01
## Summary

update https://www.ubuntu.com/legal/ubuntu-advantage/service-description per the [copy doc](https://docs.google.com/document/d/1mrNrsC1WDf6QTg3nrVCEZS8vJzGz1fokvoFDVC4s_mI/edit)

### LINKED ISSUE #2749: UI Issues on BootStack service description page
These are known issues, but creating an issue to make sure it's logged. There are UI issues on the BootStack service description page.

Issues include:
- List number is a different size than the text next to it
- List wrapping vs non-list wrapping is different
- Severity Level definitions shouldn't be numbered (i.e. "4.1 Severity Level 1 — Core functionality not available" should not include the 4.1)
- Severity Level definitions "headers" should be a more normal size ("Severity Level 1 — Core functionality not available"), similar to UASD
---
*Reported from: https://www.ubuntu.com/legal/bootstack/service-description*

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Updated Ubuntu Advantage service description
- Made Appendix 1 an include
- Used includes from the BootStack service description
- Fixed some of the css issues for font size of number bullets and indents of paragraphs, plain lists and tables

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [service description](http://0.0.0.0:8001//legal/ubuntu-advantage/service-description)
- compare to the [copy doc](https://docs.google.com/document/d/1mrNrsC1WDf6QTg3nrVCEZS8vJzGz1fokvoFDVC4s_mI/edit#)
- ask @chrisjohnston about the position of the ‘Severity levels and target response times’ include

## Issue / Card

Fixes #2748 and partially fixes #2749

### CODE REVIEW

**Comment by frankiegranato** [comment]
Oh dear @pmahnke - I can't seem to see the changes in the copydoc's history anymore so I can't really "check" that they've all been implemented. Do you think I need to?

**Comment by pmahnke** [comment]
@frankiegranato normally I tick them off as I go... I have re-opened them all for you to do the same... you can too

image

**Comment by cjohnston1158** [comment]
Missing DSE from the TOC

File: templates/legal/shared/_appendix-support-scope.html [hunk 1]
```diff
@@ -0,0 +1,692 @@
...
+              <td class="u-align--center"><img width="16" height="16" alt="Yes" src="{{ ASSET_SERVER_URL }}c4b02d61-tick-orange.svg" /></td>
+            </tr>
+            <tr>
+              <td>Unlimited <a href="#kvm-guest-support">Ubuntu KVM guest support</a></td>
+              <td></td>
+              <td></td>
+              <td class="u-align--center"><img width="16" height="16" alt="Yes" src="{{ ASSET_SERVER_URL }}c4b02d61-tick-orange.svg" /></td>
+            </tr>
+            <tr>
+              <td><a href="#kvm-guest-support">FIPS</a>-certified cryptographic modules</a>
```
Comment (anthonydillon) [inline templates/legal/shared/_appendix-support-scope.html]: Strap `</a>` at the end.

**Comment by pmahnke** [comment]
@chrisjohnston do you want to look again?

**Comment by cjohnston1158** [comment]
The specific issues mentioned have been addressed.

Still -1 on introducing the UI issues into the UASD.

**Comment by pmahnke** [comment]
@cjohnston1158 has +1'ed this

**Comment by cjohnston1158** [comment]
+1

Thank you for your work on this Peter!

**Pull request merged**